src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -109,6 +109,18 @@ const en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonContent) => ({
     buttonText: 'Apply for final order',
     buttonLink: `${isApplicant2 ? `${APPLICANT_2}${FINALISING_YOUR_APPLICATION}` : FINALISING_YOUR_APPLICATION}`,
   },
+  awaitingJointFinalOrderOrFinalOrderOverdue: {
+    line1: `Your ${partner} has not yet applied for a final order. They also have to apply so your ${
+      isDivorce ? 'divorce application' : 'to end your civil partnership'
+    } can be finalised jointly.`,
+    subHeading: 'What you can do',
```

Comment:
Can we re-use the same line here: https://github.com/hmcts/nfdiv-frontend/blob/fd7a0a104e434402c43965384f92dc90f0d8a995/src/main/steps/applicant1/hub-page/joint/content.ts#L79

Reply:
Looking at the wording for line1 there, I think that needs to change.  It's fine for divorce, but if the citizen is in a CP the text is a mess and doesn't make sense.

Reply:
I just mean we're reusing the subheading 'What you can do' so can we make this a common element shared between conditionalOrderPending.afterDueDate: and awaitingJointFinalOrderOrFinalOrderOverdue:
it'll probs be used again in the future too

Reply:
Will address this otherwise.